### ISSUE TITLE
Build alt text into ggplot objects

### ISSUE THREAD

**Issue by MargaretSiple-NOAA** [alaska-groundfish-efh/EFHSDM#12]
Figure out how to make alt text in ggplot built into ggplot object (is this already possible?). 

Later: See [BrailleR package](https://github.com/ajrgodfrey/BrailleR). Kelli Johnson et al have a grant to crowdsource what alt text should be for figures. This is coming in the future but will be super helpful.

**Comment by MargaretSiple-NOAA** [comment]
Merging into #11
